.gitignore

```diff
@@ -1,3 +1,4 @@
 obj/
 bin/
 *.user
+artifacts/
```

Comment:
This is needed only for testing, right? Or are the build artifacts committed to the repo somehow? Same for HE.

Reply:
Yes, this is just to do local testing (quite common to have it in .gitignore though)